showroom.js

```diff
@@ -0,0 +1,69 @@
+//Problem : Create a application for a showroom showing 3 types of car having some common data names and some uncommon data names. Use inheritance to accomplish it.
+var Vehicle = function(data) {
+  this.name = data.name;
+  this.model = data.model;
+  this.make = data.make;
```

Comment:
@abansal26 Please use the specific variable name. What 'make' specify here?

Reply:
changed make with makingyear in new commit. Commit id - 440cb92c8777dffa92ac0868ee94aa4ef8b8d84c